app/controllers/assessments_controller.rb

```diff
@@ -150,10 +149,20 @@ def update
     # Figure out the change
     delta = []
     typed_reported_symptoms.each do |symptom|
-      new_val = symptom.bool_value
-      old_val = assessment.reported_condition&.symptoms&.find_by(name: symptom.name)&.bool_value
-      if new_val.present? && old_val.present? && new_val != old_val
-        delta << symptom.name + '=' + (new_val ? 'Yes' : 'No')
+      case symptom.type
```

Comment:
Now that you have separated this change into a separate PR - can you remove it from this branch/PR?